recipes/renode-cli/recipe.yaml

```diff
@@ -0,0 +1,86 @@
+context:
+  name: renode-cli
+  version: "1.15.3"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/renode/renode/releases/download/v${{ version }}/renode_${{ version }}_source.tar.xz
+    sha256: 40c50afe8db86a9c63f4a86b93116a6babd43bf241cfaac47b2c0d2cbc2b15a8
+    patches:
+      - patches/update-IronPython-Renode_NET.csproj.patch
+      - patches/update-cores.template_NET.csproj.patch
+      - patches/update-IronPython-Unittest_NET.csproj.patch
+      # - patches/fix-from-master-Renode_NET.sln.patch # rattler seems to choke on leading tabs
+      - patches/fix-from-master-TermSharp_NET.csproj.patch
+      - patches/fix-from-master-Xwt.Gtk3_NET.csproj.patch
```

Comment:
Is this patch still needed since we are building without the GUI?

Reply:
It is needed, it caused a fail at my end, I moved it to the script